scripts/start_port.sh

```diff
@@ -82,5 +108,14 @@ git commit \
 
 set +x
 
-echo "After pushing, you can open a PR at:"
-echo "https://github.com/leanprover-community/mathlib4/compare/$branch_name?expand=1&title=feat:+port+$mathlib4_mod_tail&labels=mathlib-port"
+if [ "$RESTART" ]; then
+    echo "# The script just created a branch $branch_name. You may want to:"
+    echo "git checkout $old_branch_name"
+    echo "git fetch"
+    echo 'git merge $(git merge-base '"$old_branch_name"' origin/master)'
```

Comment:
I changed this from a rebase to a merge, since a merge is less likely to be annoying

Reply:
But a rebase is better for reviewing, isn't it?

Reply:
The `git reset --soft` on the next line throws away all the history up to this point anyway.

Reply:
* The `tmp` branch grabs data from `mathlib3port`. It doesn't need to have an up-to-date `master` for `mathlib4`.

Reply:
> Why `git merge-base`? Isn't this merge a no-op?

Whoops, this should be
```suggestion
    echo 'git merge $(git merge-base '"$branch_name"' origin/master)'
```

Reply:
Why not just drop `git fetch`?

Reply:
Because if the user runs git fetch anyway then the advice remains incorrect.